schemas/com.vimeo/chapter_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Vimeo Chapter.",
+  "self": {
+    "vendor": "com.vimeo",
+    "name": "chapter_event",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "index": {
+      "type": [
+        "integer",
+        "null"
+      ],
+      "description": "The chapter number.",
+      "maximum": 100,
+      "minimum": 0
+    },
+    "startTime": {
+      "type": [
+        "integer",
+        "null"
+      ],
+      "description": "The time in seconds when the chapter begins.",
+      "maximum": 2147483647,
+      "minimum": 0
+    },
+    "title": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "The chapter title.",
+      "maxLength": 50
```

Comment:
Not sure if it's relevant, but [here I found](https://stackoverflow.com/questions/58269596/what-is-the-maximum-length-for-vimeo-titles-and-descriptions) that the title max length can be 128 chars, but even that seems like living on the edge a bit, we have 4096 chars for the `label` property max length. Do you have any guidance from some other docs what could the limit be?

Reply:
I can't find any other docs on the max length (though it is likely 128 from a few other search results?), 4096 seems like a sensible `maxLength` to me though.